FogCreek/Glitch-Community
# Pop-nav: update PopoverSearch component to use shared components

## Links
* pop-nav.glitch.me
* [Clubhouse](https://app.clubhouse.io/glitch/story/1685/a11y-add-to-collection-pop-keyboard-navigation)
* [Keyboard navigation spec](https://docs.google.com/document/d/1oRRujFM4Z1rLhlgyTif3HOtRoFSBU4xutvQ_R33AkWc/edit?usp=sharing) 

## GIF/Screenshots:
![image](https://user-images.githubusercontent.com/4480480/69286674-7485c300-0bb9-11ea-91b5-75fa3ea6ad79.png)

![image](https://user-images.githubusercontent.com/4480480/69286815-de9e6800-0bb9-11ea-919e-c16f0f985016.png)


## Changes:
* Updates the PopoverSearch component to use the shared ResultsList component (and its friends) 
  * changes the keyboard navigation on these components: tab from the filter list to the first result, then use arrow keys to move up and down in the results list
  * removes the 'View' button from the add to collection popover results
* Changes the dropdown project list on the play collection page to use the styling from shared components instead of overridden styles. 

## How To Test:
* Try out the various popovers using both keyboard and mouses. 
 * See: dropdown to select a project on the team analytics section, add to collection popovers, add project to team, add user to team popovers. 

## Things left to do: 
@ehmorris has a PR for shared-components that standardizes/adds in hover styling on the ResultsList elements; we should update shared components & this PR before releasing this.
